opal/mca/accelerator/null/accelerator_null_component.c

```diff
@@ -393,3 +395,12 @@ static int accelerator_null_get_mem_bw(int device, float *bw)
     *bw = 1.0; // return something that is not 0
     return OPAL_SUCCESS;
 }
+
+static void mca_accelerator_cuda_ze_memkind (ompi_memkind_t *memkind)
+{
+  memkind->im_name = NULL;
+  memkind->im_no_restrictors = false;
+  memkind->im_num_restrictors = 0;
+
+  return;
+}
```

Comment:
```suggestion
static void mca_accelerator_null_memkind (ompi_memkind_t *memkind)
{
  memkind->im_name = NULL;
  memkind->im_no_restrictors = false;
  memkind->im_num_restrictors = 0;

  return;
}
```

We don't need to return `system` here, right?

Reply:
I also don't see where this callback is used

Reply:
the 'system' and 'mpi' memkinds are set in the info_memkind.c, independent of the accelerator component used.